packages/docusaurus-theme-classic/src/theme/SkipToContent/index.tsx

```diff
@@ -14,7 +14,11 @@ import styles from './styles.module.css';
 export default function SkipToContent(): JSX.Element {
   const {containerRef, handleSkip} = useSkipToContent();
   return (
-    <div ref={containerRef} role="region">
+    <div ref={containerRef} role="region" aria-label={<Translate
+          id="theme.common.skipToMainContentAriaLabel"
+          description="The skip to content aria label used for accessibility of the region div ">
+          Skip to main content
+        </Translate>}>
```

Comment:
You should use `translate`. JSX elements are objects, not strings. You can check the other `aria-label`s:

https://github.com/facebook/docusaurus/blob/94067ce9305653a4534f23c96406ae7fe9e35837/packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/CollapseButton/index.tsx#L25-L29

It also needs to be formatted. If you don't want to check this out locally and run format, I can do that for you later.

Reply:
fixed. Thanks for the help. I'm not really a JS pro...